actors/power/src/lib.rs

```diff
@@ -316,6 +316,13 @@ impl Actor {
         Ok(MinerConsensusCountReturn { miner_consensus_count: st.miner_above_min_power_count })
     }
 
+    fn ramp_duration(rt: &impl Runtime) -> Result<u64, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let st: State = rt.state()?;
+        
+        Ok(st.ramp_duration_epochs)
+    }
+
```

Comment:
I think the ramp configuration should be rolled into the return value from the existing request for total power. All the call sites above that called this function also invoked the power actor for the total power just a few lines earlier.

Reply:
Thanks - that makes sense and reduces the number of overall changes, I've updated accordingly.